libs/architectures/pyproject.toml

```diff
@@ -14,8 +14,7 @@ h5py = "^3.9.0"
 [[tool.poetry.source]]
 name = "torch"
 url = "https://download.pytorch.org/whl/cu118"
-secondary = true
-default = false
+priority = "explicit"
```

Comment:
I'm guessing this syntax got depreated?

Reply:
Yeah, this is the syntax as of 1.5. It also stops all of those spurious "authorization" warnings, which is nice